src/check.py

```diff
@@ -1,3 +1,26 @@
 """
 This file includes stuff actual checking
-"""
\ No newline at end of file
+"""
+from typing import List
+from math import log10 as log
+
+
+class PlagiarismCheck:
+    def __init__(self, index: Index, query_path: str):
+        self.index = index
+        self.query_path = query_path
```

Comment:
You will be given the document contents themselves. Not the path to the document (the contents could be given on stdin too).